pokemongo_bot/walkers/polyline_walker.py

```diff
@@ -15,6 +15,6 @@ def __init__(self, bot, dest_lat, dest_lng):
                                                               (self.dest_lat, self.dest_lng),
                                                               self.speed, google_map_api_key=self.bot.config.gmapkey)
         self.pol_lat, self.pol_lon = self.polyline.get_pos()
-        self.pol_alt = self.polyline.get_alt() or uniform(self.bot.config.alt_min, self.bot.config.alt_max)
+        self.pol_alt = self.polyline.get_alt() or self.alt
```

Comment:
There is already PR https://github.com/PokemonGoF/PokemonGo-Bot/pull/5135 for this